pbs/pbsrequest.go

```diff
@@ -20,6 +20,13 @@ import (
 
 const MAX_BIDDERS = 8
 
+type MediaType byte
+
+const (
+	MEDIA_TYPE_BANNER MediaType = iota
+	MEDIA_TYPE_VIDEO
```

Comment:
In go, I believe the convention for constants is to use camel case because if all constants were uppercase then they would all be global. I know that the codebase's existing constants are all uppercase. Do you think it makes sense to add these new constants in camel case?

Reply:
I would think, for consistency sake, let's have it uniform. We can do another change later and change case for all constants in the codebase.